.github/workflows/nv-sd.yml

```
@@ -0,0 +1,54 @@
+name: nv-a6000
+
+on:
+  schedule:
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
```

```suggestion
  schedule:
    - cron: "0 0 * * 0"
  workflow_dispatch:
  pull_request:
    paths:
      - "deepspeed/ops/transformer/inference/diffusers_**"
      - "tests/unit/inference/test_stable_difussion.py"
```

Something like this should make the test trigger anytime there are changes to Stable Diffusion code/tests (but there may be additional files to add)

This should make the test trigger

---

Updated, thanks.